src/node/snapshotter.h

```diff
@@ -327,11 +336,21 @@ namespace ccf
         idx,
         next_snapshot_indices.front());
 
-      auto snapshot_idx = next_snapshot_indices.front();
-      if (snapshot_idx - last_snapshot_idx >= snapshot_tx_interval)
+      bool forced_snapshot = idx > forced_snapshot_after_idx &&
```

Comment:
I don't see `forced_snapshot_after_idx` being set to anything apart from `-1` here?